packages/spaces/src/Spaces.js

```diff
@@ -21,21 +21,24 @@ export const getAllSpaces = async (
 
   const {
     data: {
-      data: { spaces },
+      data: { configurationPagination },
     },
-  } = await avSlotMachineApi.create(
+  } = await avWebQLApi.create(
     {
       query,
       variables,
     },
     { headers: { 'X-Client-ID': clientId } }
   );
 
-  const { totalCount, page, perPage } = spaces;
-  const unionedSpaces = _spaces.concat(spaces.spaces);
+  const {
+    pageInfo: { itemCount, currentPage, perPage },
+  } = configurationPagination;
+
+  const unionedSpaces = _spaces.concat(configurationPagination.items);
 
-  if (totalCount > page * perPage) {
-    const vars = { ...variables, page: page + 1 };
+  if (itemCount > currentPage * perPage) {
```

Comment:
`pageInfo` contains a `hasNextPage` boolean we could use here for readability, but I'm fine with this either way

Reply:
good idea. i left as a TODO for now f70a632